src/site/content/en/blog/badging-api/index.md

```diff
@@ -69,15 +66,15 @@ Examples of sites that may use this API include:
 | ------------------------------------------ | ---------------------------- |
 | 1. Create explainer                        | [Complete][explainer]        |
 | 2. Create initial draft of specification   | [Complete][spec]             |
-| 3. Gather feedback & iterate on design     | [In progress](#feedback)     |
-| **4. Origin trial**                        | v2 (Chrome 79-81) [In progress](#ot)<br>v1 (Chrome 73-78) [In progress](#backwards-compat) |
-| 5. Launch                                  | Not started                  |
+| 3. Gather feedback & iterate on design     | Complete                     |
+| 4. Origin trial                            | Complete                     |
+| **5. Launch**                              | Chrome 82                    |
```

Comment:
It's going out in 81.

Reply:
Badging first landed in 82, and was later merged into 81. Maybe that's why 82 is mentioned. So yes, badging is enabled in 81 and is not experimental.